Bash expects an error message, but Solaris has slightly differently worded one.
This patches the expected test output to pass on Solaris.

This is not suitable for upstream.

--- tests/vredir.right
+++ tests/vredir.right
@@ -91,7 +91,7 @@ swizzle ()
 }
 10 11
 10 11
-./vredir4.sub: line 51: ${output}: Bad file descriptor
+./vredir4.sub: line 51: ${output}: Bad file number
 12 10
 a
 a
@@ -119,5 +119,5 @@ swizzle ()
     exec {stdout}>&${fd[1]}-
 }
 redir 2
-./vredir8.sub: line 33: $fd: Bad file descriptor
-./vredir8.sub: line 38: $fd: Bad file descriptor
+./vredir8.sub: line 33: $fd: Bad file number
+./vredir8.sub: line 38: $fd: Bad file number
